spec/features/lettings_log_spec.rb

```diff
@@ -110,15 +211,115 @@
+
+      context "when the current user's organisation doesn't hold stock" do
+        it "shows the managing organisation question" do
+          user.organisation.update!(holds_own_stock: false)
+          visit("/lettings-logs")
+          click_button("Create a new lettings log")
+          click_link("Set up this lettings log")
+          log_id = page.current_path.scan(/\d/).join
+          if page.current_path == "/lettings-logs/#{log_id}/owning-organisation"
+            click_link("Skip for now")
+          end
+          expect(page).to have_current_path("/lettings-logs/#{log_id}/managing-organisation")
+          select(user.organisation.name, from: "lettings-log-managing-organisation-id-field")
+          click_button("Save and continue")
+          visit("lettings-logs/#{log_id}/setup/check-answers")
+          expect(page).to have_content("Managing agent User org")
+        end
+      end
+
+      context "when the current user's organisation does hold stock" do
+        before do
+          user.organisation.update!(holds_own_stock: true)
+        end
+
+        context "and the user's organisation has no managing agents" do
+          it "doesn't show the managing organisation question" do
+            visit("/lettings-logs")
+            click_button("Create a new lettings log")
+            click_link("Set up this lettings log")
+            log_id = page.current_path.scan(/\d/).join
+            if page.current_path == "/lettings-logs/#{log_id}/owning-organisation"
+              click_link("Skip for now")
+            end
+            expect(page).not_to have_current_path("/lettings-logs/#{log_id}/managing-organisation")
+            visit("lettings-logs/#{log_id}/setup/check-answers")
+            expect(page).not_to have_content("Managing agent ")
+          end
+        end
+
+        context "and the user's organisation has 1 or more managing agents" do
+          let!(:managing_org) { create(:organisation, name: "Managing org") }
+          let!(:org_rel) { create(:organisation_relationship, parent_organisation: user.organisation, child_organisation: managing_org) }
+
+          it "does show the managing organisation question" do
+            visit("/lettings-logs")
+            click_button("Create a new lettings log")
+            click_link("Set up this lettings log")
+            log_id = page.current_path.scan(/\d/).join
+            if page.current_path == "/lettings-logs/#{log_id}/owning-organisation"
+              click_link("Skip for now")
+            end
+            expect(page).to have_current_path("/lettings-logs/#{log_id}/managing-organisation")
+            select(managing_org.name, from: "lettings-log-managing-organisation-id-field")
+            click_button("Save and continue")
+            visit("lettings-logs/#{log_id}/setup/check-answers")
+            expect(page).to have_content("Managing agent Managing org")
+            expect(user.organisation.managing_agents).to eq(org_rel.child_organisation)
```

Comment:
Similarly to above, this line is pretty artificial/contrived way of explicitly referencing the organisation relationship in the test. Without it, Rubocop complains that something being defined with `let!` is not being explicitly used in the test. The reason I have defined the organisation relationship at all is because its existence influences the logic around the answer options for the question, and in particular whether the question gets auto-assigned an answer.